community/getHelp.html.haml

```diff
@@ -86,30 +86,20 @@ title: Get help
   .col-md-12
     :asciidoc
       [#chat]
-      == Chat
+      == Chat real-time
 
-      Got questions? Want to discuss an issue in real-time? Then join us on IRC.
+      To follow our development discussions, join us on https://kie.zulipchat.com[Zulip chat].
 
-      Most of our project discussion take place on IRC, usually during European and American daytime.
+      - If you're working on a contribution, come talk to us on `#jbpm` stream.
+      The core developers are there during working hours (depending on their timezone).
 
-      === Connection info
+      Most of our project discussion take place on real-time chat, usually during European and American daytime.
```

Comment:
Just a nit-picking, should be `takes place`.

Reply:
Fair point this sentence was originally there just moved, but then I realised it's discussion missing the s, discussions.